storybook/src/css/Button.css

```diff
@@ -0,0 +1,33 @@
+.custom-button {
+    border: none;
+    padding: 12px 20px;
```

Comment:
Absolute padding can be pretty bad, make sure that this scales for different screen sizes.